source/php-frameworks/symfony.txt

```diff
@@ -0,0 +1,372 @@
+.. _php-symfony-integration:
+
+===========================
+Symfony MongoDB Integration
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, doctrine odm, web app
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the Symfony MongoDB integration and
+how to use this framework to build a simple PHP web application. You can
+learn about the benefits of using Symfony to build
+web applications with MongoDB as your database, as well as libraries
+that you can leverage to simplify querying MongoDB.
+
+Symfony is a flexible and highly configurable framework for building PHP
+applications. You can use this framework to create reusable components
+to streamline your web app.
+
+The :ref:`php-symfony-qs` section of this guide contains a tutorial
+which you can follow to build a single page app that accesses data from
+a MongoDB collection.
+
+The :ref:`php-symfony-resources` section contains links to resources and
+documentation for further learning.
+
+Why Use MongoDB in a Symfony Application?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By using MongoDB as a data store in a Symfony web application, you can
+leverage the document data model to build rich query expressions,
+allowing you to easily interact with data. You can also customize your
+connections to persist and access data to and from multiple databases
+and collections.
+
+In your applications, you can implement the **MongoDB Doctrine ODM**,
```

Comment:
```suggestion
In your applications, you can implement the **Doctrine MongoDB ODM**,
```

Consistent with the project's own naming (see: [docs](https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/)).

I corrected various other references that follow, but you might alternatively prefer to just shorten those references to "Doctrine ODM".